v2/devices/FP4.yml

```diff
@@ -0,0 +1,198 @@
+name: "Fairphone 4"
+codename: "FP4"
+formfactor: "phone"
+aliases: ["fp4"]
+doppelgangers: []
+user_actions:
+  confirm_model:
+    title: "Confirm your model"
+    description: "Please check that your device is a Fairphone 4 (FP4)."
+  confirm_os:
+    title: "Confirm OS version"
+    description: "Your device must be running the Fairphone OS version of Android 11 before installing Ubuntu Touch. With a previously installed custom ROM, it won't work!"
+    link: "https://support.fairphone.com/hc/en-us/articles/4405858261777?s=09"
+  bootloader:
+    title: "Reboot to Bootloader"
+    description: "With the device powered off, press and hold the VOLUME DOWN button and plug the device into your PC via USB. After some seconds you'll see the fastboot mode."
+    image: "phone_power_down"
+    button: true
+  bootloader_from_fastbootd:
+    title: "Reboot to Bootloader"
+    description: 'With the device still at the fastbootd screen, use the volume buttons to select "Reboot to bootloader".'
+    image: "phone_power_down"
+    button: true
+  recovery:
+    title: "Reboot to Recovery"
+    description: 'With the device still at the "FastBoot Mode", use the Volume buttons to switch to "Recovery Mode" and push the power button to confirm your selection.'
+    image: "phone_power_up"
+    button: true
+  fastbootd:
+    title: "Reboot to fastbootd mode"
+    description: 'With the device still at the "FastBoot Mode", use the Volume buttons to switch to "Recovery Mode" and push the power button to confirm your selection. Then select "Fastboot mode" from the list.'
+    image: "phone_power_up"
+    button: true
+unlock:
+  - "confirm_model"
+  - "confirm_os"
+handlers:
+  bootloader_locked:
+    actions:
+      - fastboot:oem_unlock:
+          code_url: "https://www.fairphone.com/en/bootloader-unlocking-code-for-fairphone-3"
+operating_systems:
+  - name: "Ubuntu Touch"
+    compatible_installer: ">=0.9.2-beta"
```

Comment:
please set this to: `compatible_installer: ">=0.9.6-beta"` as it will require installer version `0.9.6` and above. :)